tests/integration/CoreApiIntegrationTest.php

```diff
@@ -112,4 +112,111 @@ public function testChargeGopay()
         $this->assertEquals('201', $this->charge_response->status_code);
         $this->assertEquals('pending', $this->charge_response->transaction_status);
     }
+
+    public function testCreatePayAccount()
+    {
+        $params = array(
+            "payment_type" => "gopay",
+            "gopay_partner" => array(
+                "phone_number" => 874567446788,
+                "redirect_url" => "https://www.google.com"
+            )
+        );
+        $this->charge_response = CoreApi::linkPaymentAccount($params);
+        $this->assertEquals('201', $this->charge_response->status_code);
+        $this->assertEquals('PENDING', $this->charge_response->account_status);
+    }
+
+    public function testGetPaymentAccount()
+    {
+        try {
+            $this->charge_response = CoreApi::getPaymentAccount("dummy");
+        } catch (\Exception $e) {
+            $this->assertContains("Midtrans API is returning API error.", $e->getMessage());
+        }
+    }
+
+    public function testUnlinkPaymentAccount()
```

Comment:
On test names which result in negative-scenario e.g: 404 better to give it a more descriptive name like `testFail404ToUnlinkPaymentAccountWithNonExistAccount` or `testUnlinkPaymentAccountWithNonExistAccount`

Use `testUnlinkPaymentAccount` test name for scenario that return 200 success